ShapeAreaCalculator.java

```diff
@@ -0,0 +1,82 @@
+package tasks;
+
+/**
+ * This program demonstrates the abstraction and calculates the area of different shapes.
+ * 
+ * @author Jayesh Soni
+ * @since 2024-04-22
+ */
+
+/**
```

Comment:
Above the Class only one documentation comment or anytime Need to add only one documentation comment All thing you can add in that. You also can you
Tag for the new line and another some html tag also